pkg/scheduler/apis/config/validation/validation_test.go

```diff
@@ -216,7 +217,7 @@ func TestValidateKubeSchedulerConfigurationV1beta2(t *testing.T) {
 	scenarios := map[string]struct {
 		expectedToFail bool
```

Comment:
can we get rid of this boolean?
`cmp.Diff` already checks if the errors matches the empty list.

Reply:
I think so. I'll remove it.

Reply:
Deleted all `expectedToFail` bool and check ``errs == nil && diff !="" `  for unexpected success instead.